worlds/witness/place_early_item.py

```diff
@@ -0,0 +1,149 @@
+from logging import debug, error
+from typing import TYPE_CHECKING, Dict, List, Set, Tuple
+
+from BaseClasses import CollectionState, Item, Location, LocationProgressType
+
+from .data import static_logic as static_witness_logic
+
+if TYPE_CHECKING:
+    from . import WitnessWorld
+
+
+def get_available_early_locations(world: "WitnessWorld") -> List[Location]:
+    # Pick an early item to put on Tutorial Gate Open.
+    # Done after plando to avoid conflicting with it.
+    # Done in fill_hook because multiworld itempool manipulation is not allowed in pre_fill.
+
+    # Prioritize Tutorial locations in a specific order
+    tutorial_checks_in_order = [
+        "Tutorial Gate Open",
+        "Tutorial Back Left",
+        "Tutorial Back Right",
+        "Tutorial Front Left",
+        "Tutorial First Hallway Straight",
+        "Tutorial First Hallway Bend",
+        "Tutorial Patio Floor",
+        "Tutorial First Hallway EP",
+        "Tutorial Cloud EP",
+        "Tutorial Patio Flowers EP",
+    ]
+    available_locations = [
+        world.get_location(location_name) for location_name in tutorial_checks_in_order
+        if location_name in world.reachable_early_locations  # May not actually be sphere 1 (e.g. Obelisk Keys for EPs)
+    ]
+
+    # Then, add the rest of sphere 1 in "game order"
+    available_locations += sorted(
+        (
+            world.get_location(location_name) for location_name in world.reachable_early_locations
+            if location_name not in tutorial_checks_in_order
+        ),
+        key=lambda location_object: static_witness_logic.ENTITIES_BY_NAME[location_object.name]["processing_order"]
+    )
+
+    return [
+        location for location in available_locations
+        if not location.item and not location.progress_type == LocationProgressType.EXCLUDED
```

Comment:
```suggestion
        if not location.item and location.progress_type != LocationProgressType.EXCLUDED
```

Reply:
Fair

Idk why I like `not a == b` so much more than `a != b`, the latter is definitely better but I have a weird attachment to the former lol